CONTRIBUTING.md

```diff
@@ -136,6 +137,47 @@ cases to our [automated test suite](ci/README.md) to guarantee enough
 coverage. A PR that makes significant code changes without contributing new test
 cases will be flagged by reviewers and will not be accepted.
 
+### Getting your PR verified by CI
```

Comment:
I think this information could be moved to `ci/README.md` or at the very least I feel like we should unify this section with the existing `CI testing` section. What do you think?

Reply:
I have merged these two sections into one.

I prefer to put these trigger phrases in `CONTRIBUTING.md`, which I think is a must read for developers. While `ci/README.md` is optional and more detailed (how CI works).